proprietary/tokens/src/common/amsterdam/hyphenation.tokens.json

```diff
@@ -0,0 +1,10 @@
+{
+  "amsterdam": {
+    "hyphenation": {
+      "hyphenate-limit-chars": { "value": "12 8 4" },
+      "hyphens": { "value": "auto" },
+      "overflow-wrap": { "value": "break-word" },
+      "word-break": { "value": "auto-phrase" }
```

Comment:
This only works for Japanese? Doesn't seem very necessary imo

Reply:
No indeed, removed.

Reply:
I’ve removed it.